build_from.py

```diff
@@ -157,5 +192,12 @@ def build_by_notice(filename, title, act_title, act_section,
         build_by_notice(args.filename, args.title, args.act_title,
                         args.act_section, args.notices_to_apply,
                         args.last_notice, args.checkpoint)
+
+    elif args.operation == 'generate_xml':
+
+        generate_xml(args.filename, args.title, args.act_title,
+                     args.act_section, args.notices_to_apply,
+                     args.last_notice, args.checkpoint)
+
```

Comment:
Just a place-holder here to note that @cmc333333 was working on a generic `eregs` script to run all the things. We'd want to work this into that. Nothing particular to change here yet though.